templates/ts/ai-assistant-bot/infra/azure.bicep

```diff
@@ -60,11 +59,15 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
         }
         {
           name: 'BOT_ID'
-          value: botAadAppClientId
+          value: identity.properties.clientId
+        }
+        {
+          name: 'BOT_TENANT_ID'
+          value: identity.properties.tenantId
         }
         {
-          name: 'BOT_PASSWORD'
-          value: botAadAppClientSecret
+          name: 'BOT_TYPE'
+          value: 'UserAssignedMsi'
```

Comment:
Any feature difference between `UserAssignedMsi` and `MultiTenant`?
E.g., use bot in another tenant? Or SSO? Or any scenario that only work for multi tenant?

Reply:
The `UserAssignedMsi` type bot can also be used for users from different tenant.